packages/api-gateway/.eslintrc.js

```diff
@@ -32,4 +32,20 @@ module.exports = {
       },
     ],
   },
+  plugins: ['import'],
```

Comment:
Why do we need this? I never had any issues importing these packages.
As long as you have the reference to these packages in the `packages/api-gateway/package.json` should be fine.

Did you try to run `yarn bootstrap`?

Reply:
Can you remove this configuration? I tested your PR and everything is working fine without this configuration